zipkin-storage/cassandra/src/main/java/zipkin/storage/cassandra/CompositeIndexer.java

```diff
@@ -38,8 +38,6 @@
         CacheBuilder.from(spec).<PartitionKeyToTraceId, Pair<Long>>build().asMap();
     Indexer.Factory factory = new Indexer.Factory(session, indexTtl, sharedState);
     this.indexers = ImmutableSet.of(
-        factory.create(new InsertTraceIdByServiceName(bucketCount)),
-        factory.create(new InsertTraceIdBySpanName()),
         factory.create(new InsertTraceIdByAnnotation(bucketCount))
```

Comment:
since this makes the indexer thing really only used for a single table, now, you can put in a TODO for me to cleanup and simplify this